internal/pkg/plugin/trellogithub/create.go

```diff
@@ -24,11 +24,14 @@ func Create(options map[string]interface{}) (map[string]interface{}, error) {
 		}
 	}
 	log.Success("Adding workflow file succeeded.")
-	trelloIds, err := tg.CreateTrelloItems()
-	if err != nil {
-		return nil, err
+
+	trelloIds := &TrelloItemId{
+		boardId:     tg.GetApi().BoardId,
```

Comment:
I think "GetApi" is a little ambiguous. What does it get exactly? An API client?

Based on the code, I think it returns a struct, which contains some information, which are board ID, list IDs, etc. They have nothing to do with "API". I think we need a better naming here.

Reply:
Also, where does the boardID, list IDs come from?

It seems they are part of the "options", but the documentation isn't updated. The doc should include an example, letting users know what parameters are required in the "options" section.

An example of how to refer to trello outputs is also required. Refer to my previous pull request for an example: https://github.com/merico-dev/stream/pull/316/files.

Reply:
1. will modify 
2. boardID, list IDs come from ${{xxx}}.  will update the doc